storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11973,23 +11973,30 @@ void rocksdb_set_collation_exception_list(THD *const thd,
   *static_cast<const char **>(var_ptr) = val;
 }
 
-void rocksdb_set_bulk_load(THD *const thd, struct st_mysql_sys_var *const var
-                                               MY_ATTRIBUTE((__unused__)),
-                           void *const var_ptr, const void *const save) {
-  Rdb_transaction *&tx = get_tx_from_thd(thd);
+int rocksdb_check_bulk_load(THD *const thd,
+                            struct st_mysql_sys_var *var
+                                MY_ATTRIBUTE((__unused__)),
+                            void *save,
+                            struct st_mysql_value *value) {
+    Rdb_transaction *&tx = get_tx_from_thd(thd);
```

Comment:
Nit: These look like whitespace changes. It should be two spaces per indent.

Reply:
Oops, sorry, fat-finger on my part. Will be fixed.